module/VuFind/tests/unit-tests/src/VuFindTest/RecordDriver/EDSTest.php

```diff
@@ -571,15 +571,34 @@ public function testGetAllSubjectHeadingsFlattened(): void
         );
     }
 
+    /**
+     * Data provider for testGetThumbnail().
+     *
+     * @return array[]
+     */
+    public static function getThumbnailProvider(): array
+    {
+        return [
+            'thumb is upscaled to small' => ['small', 'small thumbnail link'],
+            'medium is used as-is' => ['medium', 'medium thumbnail link'],
+            'medium is upscaled to large' => ['large', 'medium thumbnail link'],
```

Comment:
```suggestion
            'large is downscaled to medium' => ['large', 'medium thumbnail link'],
```

Reply:
Oh, I see, you are writing from the config perspective. That setting medium will also cover the large case?